### ISSUE TITLE
Accessibility function broken in the developer tools - v. 2022.3.1    

### ISSUE THREAD

**Issue by Ralcidonis** [home-assistant/frontend#11958]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

I cannot look for an entity in dev tool using a screen reader because the HTML entity section is missing on the page.

### Describe the behavior you expected

When I use my arrow keys to navigate on the page, afer selecting the state, service,or event tab, the first section on the page (using screen reader) should have been the entity section or field. Right now, it is the "set state" section that appears first on the page. The only way to get the cursor focused on the entity field is to immediately tab around after selecting the state or service tabs. This is not normal behavior as the screen reader user would not know that the field even exist at that time.

### Steps to reproduce the issue

while using a screen reader:
1. Navigate to the dev tool.
2. Select the state or service tab with the enter key.
3. then, down arrow on the page looking for the entity section.
4. The screen reader will first locate the set state section on the page, and not the entity field section as the normal behavior.

### What version of Home Assistant Core has the issue?

core-2022.3.1

### What was the last working version of Home Assistant Core?

core-2022.2.2

### In which browser are you experiencing the issue with?

Chrome and firefox.

### Which operating system are you using to run this browser?

Win 11

**Comment by Ralcidonis** [comment]
This is still an issue that has taken accessibility backwards. It is not an enjoyable feature at the moment.
